examples/rustc-driver-interacting-with-the-ast.rs

```diff
@@ -46,7 +46,7 @@ fn main() {
         output_file: None,
         file_loader: None,
         stderr: None,
-        crate_name: None,
+        //crate_name: None,
         lint_caps: rustc_hash::FxHashMap::default(),
```

Comment:
We now need this field for more recent nightly.

```suggestion
        lint_caps: rustc_hash::FxHashMap::default(),
        parse_sess_created: None,
```

Reply:
While we're on it, could you use `rustc 1.53.0-nightly (9b0edb7fd 2021-03-27)` instead? At least, we have one breaking change, see the above.